doc/news/changes/major/20160520Jean

```diff
@@ -0,0 +1,5 @@
+Improved: The step-44 tutorial now uses the new CellDataStorage class to
```

Comment:
I suspect this file should be named something other than ` doc/news/changes/major/20160520Jean`